scripts/plugins/firewall-port

```diff
@@ -47,10 +47,19 @@ case "${OP}" in
             service iptables save
         fi
         ;;
+    check) 
+        if [[ -z `iptables -S $CHAIN | grep " $PORT "` ]]
+            then
+                echo true
+                else
```

Comment:
Indentation needs a fix. Maybe this is part of the design, but I think a more descriptive output would be better unless the output is directly used somewhere else.

Reply:
the stdout feeds into OCaml where it is fed into bool_of_string. This tells Xapi what the state of https_only should be.

Reply:
I would still use a more descriptive output and use `sscanf` to parse it on the OCaml side. That is safer than using `bool_of_string` anyway. Something like: `port 80: open` which can be easily parsed.